plugins/modules/proxmox.py

```diff
@@ -415,6 +415,18 @@
     ostemplate: 'local:vztmpl/ubuntu-14.04-x86_64.tar.gz'
     netif: '{"net0":"name=eth0,gw=192.168.0.1,ip=192.168.0.2/24,bridge=vmbr0"}'
 
+- name: Create new container with more options defining network interface with static ip4 and ip6 with vlan-tag and mtu
+  community.general.proxmox:
+    vmid: 100
+    node: uk-mc02
+    api_user: root@pam
+    api_password: 1q2w3e
+    api_host: node1
+    password: 123456
+    hostname: example.org
+    ostemplate: 'local:vztmpl/ubuntu-14.04-x86_64.tar.gz'
+    netif: '{"net0":"name=eth0,gw=192.168.0.1,ip=192.168.0.2/24,ip6=fe80::1227/64,gw6=fe80::1,bridge=vmbr0,firewall=1,tag=934,mtu=1500"}'
```

Comment:
The type of this parameter is dictionary, not string. Please write this as a dictionary instead of letting the module or ansible-core parse this string as JSON.

Reply:
i was sticking to how it was in the examples. But I've changed it now to:
```
netif:
  net0: "<string>"
```

Reply:
Thanks for also adjusting the other examples!